llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -843,6 +845,8 @@ getLineTableLengthInfoAndVersion(DWARFDataExtractor &LineTableDataReader,
     return std::move(Err);
 
   Prologue.PrologueLength = PrologueLength;
+  // DWARF 4 and below doesn't have the Address Size in the line table header,
+  // set the Offset and return.
```

Comment:
This comment feels a bit off, given how this code path is _also_ used by DWARF 5. I understand you are trying to emphasize that this field is new in dwarf 5, but we already do this on line 824 above